els/categories/urls.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework import routers
+from .api import categoryViewSet
+
+router = routers.DefaultRouter()
+router.register('api/categories', categoryViewSet, 'categories')
```

Comment:
Remove api/ as it will be handled by the parent urls.py wherein you will put the api/ prefix
```suggestion
router.register('categories', categoryViewSet, 'categories')
```